sdv/multi_table/base.py

```diff
@@ -406,6 +406,29 @@ def get_learned_distributions(self, table_name):
             f"table because it uses the '{synthesizer.__class__.__name__}'."
         )
 
+    def get_loss_values(self, table_name):
+        """Get the loss values from a model for a table.
+
+        Return a pandas dataframe mapping of the loss values per epoch of GAN
+        based synthesizers
+
+        Args:
+            table_name (str):
+                Table name for which the parameters should be retrieved.
+
+        Returns:
+            pd.DataFrame:
+                Dataframe of loss values per epoch
+        """
+        synthesizer = self._table_synthesizers[table_name]
```

Comment:
should we error if the table name is bad? I think there is an error message we use in a lot of places for that

Reply:
I think we usually use `_validate_table_name` but the full error message doesn't really make sense in this context, so I used an edited version.